### PR TITLE
Write quickstart

### PR DESCRIPTION
**Pull request by onggunhao**
* Tutorial that explains the high-level functionality of Binance Chain, and gives hands-on experience with CLI tool
* Added existing docs into the sidebar

Eventually, should be merged into the `docusaurus` branch (NOT `master`)

Preview
image

cc @huangsuyu @darren-liu @notatestuser

### CODE REVIEW

**Comment by huangsuyu** [comment]
Add how to use api to get trade info in `how to trade` part.

File: docs/guides/intro/make-trade.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+  --tif gte
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+> **Note** Quantities here are expressed without decimals, i.e. shifted by 10^8
+
+There are a few flags here that are critical:
+
+- `--side` refers to whether you are buying or selling (the Base Asset, in this case `BNB`)
```
Comment (huangsuyu) [inline docs/guides/intro/make-trade.md]: please change the explaination for `side`, 1 for buy & 2 for sell

Comment (onggunhao) [reply docs/guides/intro/make-trade.md]: Ah yes! Thanks

**Comment by onggunhao** [comment]
@huangsuyu For final approval

**Comment by huangsuyu** [comment]
> @huangsuyu For final approval

LGTM

**Pull request merged**